packages/stark-core/src/modules/error-handler.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./error-handler/actions";
```

Comment:
The parent folder of these files should be called `error-handling` instead of `error-handler`